packages/nc-gui/nuxt.config.js

```diff
@@ -90,9 +90,7 @@ export default {
   ** See https://axios.nuxtjs.org/options
   */
   axios: {
-    baseURL: process.env.NODE_ENV === 'production' ? '../' : 'http://localhost:8080/',
-    // baseURL: 'http://localhost:8080/',
-    // baseURL: 'http://localhost:8080/',
+    baseURL: process.env.NC_BACKEND_URL || 'http://localhost:8080',
```

Comment:
Keep it like this:

```
 baseURL: process.env.NC_BACKEND_URL || (process.env.NODE_ENV === 'production' ? '../' : 'http://localhost:8080/'),
```

So that even if `NC_BACKEND_URL` is not defined it will use a relative path while taking production build.